_includes/sections/browser-recommendation.html

```diff
@@ -23,10 +21,87 @@ <h1 id="browser" class="anchor"><a href="#browser"><i class="fas fa-link anchor-
 website="https://www.torproject.org/"
 forum="https://forum.privacytools.io/t/discussion-tor-browser/278"
 git="https://trac.torproject.org/projects/tor"
-windows=""
-mac=""
-linux=""
-android=""
-ios=""
-bsd=""
+windows="https://www.torproject.org/download/"
+mac="https://www.torproject.org/download/"
+linux="https://www.torproject.org/download/"
 %}
+
+<h1 id="browser-android" class="anchor"><a href="#browser-android"><i class="fas fa-link anchor-icon"></i></a> Browser Recommendations For Android</h1>
+
+{% include cardv2.html
+title="Firefox"
+image="/assets/img/tools/Firefox.png"
+description='Firefox is fast, reliable, open source and respects your privacy. Don\'t forget to adjust the settings according to our
+recommendations: <a href="#webrtc"><i class="fas fa-link"></i> WebRTC</a> and <a href="#about_config"><i class="fas fa-link"></i> about:config</a> and get the <a href="#addons"><i class="fas fa-link"></i> privacy add-ons</a>.'
+website="https://www.mozilla.org/en-US/firefox/mobile/"
+forum="https://forum.privacytools.io/t/discussion-firefox/279"
+source="https://github.com/mozilla-mobile"
+android="https://play.google.com/store/apps/details?id=org.mozilla.firefox&hl=en_US"
+%}
+
+{% include cardv2.html
+title="Tor Browser - Provides Anonymity"
+image="/assets/img/tools/Tor-Browser.png"
+description='Tor Browser is your choice if you need an extra layer of anonymity. It\'s a modified version of Firefox ESR, which comes with pre-installed privacy add-ons, encryption and an advanced proxy. <a href="https://write.privacytools.io/my-thoughts-on-security/slicing-onions-part-1-myth-busting-tor">How does Tor work?</a>'
+Website="https://torproject.org"
+forum="https://forum.privacytools.io/t/discussion-tor-browser-for-android-browsers/1522"
+git="https://gitweb.torproject.org/tor-browser.git/"
+android="https://play.google.com/store/apps/details?id=org.torproject.torbrowser&hl=en_US"
+%}
+
+{% include cardv2.html
+title="Bromite"
+image="/assets/img/tools/bromite.png"
+description="Bromite is a chromium based browser with <a href='https://www.bromite.org/#main-features'>several privacy and security related patches</a>. It also comes with an intergrated Ad blocker."
+website="https://www.bromite.org/"
+forum="https://forum.privacytools.io/t/discussion-bromite-browsers/1521"
+github="https://github.com/bromite/bromite"
+android="https://www.bromite.org/news/2018-12-21-new-fdroid-repository-url"
+%}
+
+<h3>Worth Mentioning for Android</h3>
+
+<ul>
+  <li><a href="https://www.stoutner.com/privacy-browser/">Privacy Browser</a> Privacy Browser is an open source Android web browser focused on user privacy. Note: Privacy Browser relies on the Android system webview which needs to be kept up to date to fix security issues. One can update webview by either installing it from Google Play or from the <a href='https://f-droid.org/en/packages/com.github.yeriomin.yalpstore/'>Yalp Store</a>.</li>
```

Comment:
```suggestion
  <li><a href="https://www.stoutner.com/privacy-browser/">Privacy Browser</a> Privacy Browser is an open source Android web browser focused on user privacy. <a href="https://play.google.com/store/apps/details?id=com.google.android.webview&hl=en_US"><span class="badge badge-warning" data-toggle="tooltip" title="This software relies on a closed-source central server.">WARNING: Keep Android WebView up-to-date</span> Privacy Browser relies on the Android system webview which needs to be kept up to date to fix security issues. One can update webview by either installing it from Google Play, Yalp Store or Aurora Store. It requires Android 5.0+ so using Privacy Browser on older Androids is insecure.</a></li>
```

I meant that I want to see something like this so the warning is more explicit. Warning: I haven't tested this and my wording may not be the best possible.

See also https://github.com/privacytoolsIO/privacytools.io/issues/1051#issuecomment-516255301

Reply:
looks good